src/devices/CpuTemperature/CpuTemperature.csproj

```diff
@@ -10,4 +10,8 @@
     <None Include="README.md" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Management" Version="5.0.0-alpha.1.19618.1" />

```

Comment:
I'd recommend to make it Windows-only dependency

Reply:
See above. There's no platform dependency on this dll.

Reply:
Also, any platform dependency introduced at this level would apply to the _source computer_, not the one running it.